BusBoard/BusBoard/Program.cs

```diff
@@ -12,21 +10,17 @@ public static void Main(string[] args)
         {
             while (true)
             {
-                Console.Write("Enter a bus stop ID: ");
-                var stopId = Console.ReadLine();
-
-                var tflApiUrl = "https://api.tfl.gov.uk";
+                Console.Write("Enter a postal code: ");
```

Comment:
You should consider some error handling at this point. What if the postal code they give is nonsense?